components/feature/searchwidget/src/main/res/drawable/mozac_ic_launcher_foreground.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
A-C should be "client agnostic" and not include the launcher icon of Fenix.
If this resource is really needed maybe we can set the source to be `mozac_ic_globe`.

Reply:
I added mozac_search_widget_icon that is an empty file . This file will be overridden in Focus and Fenix with the specific app icon .